compiler-rt/lib/builtins/cpu_model/x86.c

```diff
@@ -217,10 +217,11 @@ enum ProcessorFeatures {
   FEATURE_SM3,
   FEATURE_SHA512,
   FEATURE_SM4,
-  // FEATURE_APX_F,
-  FEATURE_USERMSR = 112,
-  // FEATURE_AVX10_1_256,
-  // FEATURE_AVX10_1_512,
+  //FIXME: gcc used string "apxf" instead of "egpr"
```

Comment:
The internal name is not important, so do not need a FIXME if we won't fix it.
I'm more concerning about the name in `(void)__builtin_cpu_supports("egpr");`
Does GCC use `egpr` too? Or we should change to "apxf"?

Reply:
GCC is using "apxf ". I believe we need to discuss with GCC team about this.

Reply:
Yes, the concern is from that gcc use `apxf` but not `egpr`. I don't think adding one more line like this is complete safe. I prefer to not touch APX firs and refine it in another PR, WDYT?
```
X86_FEATURE_COMPAT(EGPR,            "apxf",                   0)
```